bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -501,6 +504,10 @@ PendingReadOp parallelRead(boolean enabled) {
         return this;
     }
 
+    void allowFastFailOnBlockedServer() {
```

Comment:
"BlockedServer" is a confusing term here, since we don't really use "blocker" at any other places. I would suggest changing it to "allowFailFastOnUnwritableChannel".